src/com/ean/clarity/Main.java

```diff
@@ -0,0 +1,53 @@
+package com.ean.clarity;
+
+import com.ean.clarity.util.Baugus;
+
+import java.util.Scanner;
+
+/**
+ * Does main stuff.
+ */
+public class Main {
+
+    private static final String CLARITY_URL = "http://clarity.expedia.biz";
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Timesheet timesheet = new Timesheet();
+
+        // TODO: Change these submits to trigger on Tab key instead of Enter key (for MacBook Pro Tab8 support)
+
+        log("Monday hours: ");
+        timesheet.setMondayHours(scanner.nextInt());
+        log("Tuesday hours: ");
+        timesheet.setTuesdayHours(scanner.nextInt());
+        log("Wednesday hours: ");
+        timesheet.setWednesdayHours(scanner.nextInt());
+        log("Thursday hours: ");
```

Comment:
I don't like hardcoded week names, what happens if we add a day?  You'd have to update this.  Create an enum with valid day names.